modules/programs/zellij.nix

```diff
@@ -6,6 +6,7 @@ let
 
   cfg = config.programs.zellij;
   yamlFormat = pkgs.formats.yaml { };
+  zellijCmd = "${config.home.profileDirectory}/bin/zellij";
```

Comment:
I believe it's better to use `${lib.getExe cfg.package}`, instead of relying on a hard-coded path.

Reply:
Yes, agreed 🙂